serializations.md

```diff
@@ -945,6 +977,43 @@ The binary is a serialized Merkle Patricia Tree.
 ]
 ```
 
+### Generalized accounts
+Generalized accounts are available from version 3, Fortuna release. A
+generalized account is interchangeable with a basic (non-generalized) account
+everywhere, the only difference is that it is not able to directly sign a
+transaction. Instead the transaction has to be wrapped in a [meta
+transaction](#generalized-accounts-meta-transaction) with the correct
```

Comment:
So signed transaction and generalized accounts meta transaction are mutually exclusive?

Reply:
I wouldn't say mutually exclusive, for example state channel transactions that are co-signed could have one "normal" signature and being wrapped in one Meta. A MetaTx is the replacement for _one_ signature in the signed inner transaction!

Reply:
For co-signed transactions, can both signatures be replaced by metatx? If so, is one metatx including another metatx? If so, is there a canonical order?